x-pack/legacy/plugins/code/server/lsp/abstract_launcher.ts

```diff
@@ -161,6 +154,32 @@ export abstract class AbstractLauncher implements ILanguageServerLauncher {
     }
   }
 
+  private async doSpawnProcess(
+    installationPath: string,
+    port: number,
+    log: Logger
+  ): Promise<ChildProcess> {
+    const child = await this.spawnProcess(installationPath, port, log);
+    const pid = child.pid;
+    this.currentPid = pid;
+    this.log.debug('spawned a child process ' + pid);
+    this.spawnTimes += 1;
+    this.startTime = Date.now();
+    this.running = true;
+    if (this.options.lsp.oomScoreAdj && process.platform === 'linux') {
+      try {
+        fs.writeFileSync(`/proc/${pid}/oom_score_adj`, `${OOM_SCORE_ADJ}`);
+        this.log.debug(`wrote oom_score_adj of process ${pid} to ${OOM_SCORE_ADJ}`);
+        fs.writeFileSync(`/proc/${pid}/oom_adj`, `${OOM_ADJ}`);
```

Comment:
The ML version of this follows the `OOM_ADJ` with a `\n`, should we do the same?

Reply:
done.